SRM Elder-class Montagnes' Vessel (Glaive Replacement)

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Replaces the Glaive with the Elder-class.
![image](https://github.com/shiptest-ss13/Shiptest/assets/55075747/11ab1dd8-0227-4d6f-aa28-6fe41e9742bc)
![image](https://github.com/shiptest-ss13/Shiptest/assets/55075747/3fc417f1-6780-4a0e-b6b0-c7b83bea218a)
![elder class real](https://github.com/shiptest-ss13/Shiptest/assets/55075747/eca32c3a-a37d-4aad-838b-05dca54daef4)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The Glaive is old and a maptainer has requested I maintain my maps and replace or revamp it. I chose the former.
This version gives more attention to the non-glade sections and makes the glade less janky. This version also puts more of a focus on the non-mining and ruin running aspects of the old ship. The ashie med, botany, animals, and ashwines.
You cannot actually fish in the ship. No I will not be making it so you can. Not only would that not make sense, but it would require porting over normal fishability from the planetary turfs onto the shipturf version of water. Which I'm just not doing.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:phoaly
add: Added the Elder-class
del: Removed the Glaive-class
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
starts the process

titular

Latentish committed:
Adds the Elder Class

Adds the whole ass ship

Latentish committed:
Reflavors the Glaive files

-Replaces all mentions of the Glaive with the Elder. touched up the Elder's description.

Latentish committed:
Last touchups

- adds stuff to tables
- adds decals
- adds grasses

Latentish committed:
Fixes Roumain ID's

-Removes a couple grasses
-Removes a datum meant to assign a faction to the player. This datum actually made it so that ID's didn't get written properly, leaving players with a blank ID.

JimKil3 commented:
This is my first time reviewing a map PR, so don't take my word as gospel, lol.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/2e8b7ca8-030a-4cda-a22e-ce31beccb138)
I'm pretty sure having vents and scrubbers directly adjacent isn't best practice. I'll only use this one example, but this pops up in a lot of other places.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/c8a3e723-f0b9-4118-9bc7-b66d5050646b)
The terminal here being inaccessible is kind of weird. Maybe move the SMES off to the side, then have the PACMANs side-by-side?
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/12cfaa07-c475-4e1a-810a-8a3b8c1cd401)
Does this room need 2 vents/scrubbers? Also, this is entirely my opinion, but I'm not sure if a personal AI fits the theme of the SRM.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/eda5d9f7-9c5b-4cc4-868b-99b8a97ce8eb)
Instead of mapping in the contents like this, I'd personally add the items that are in the hunter/shadow lockers to the locker's `PopulateContents` proc - the lockers are already a subtype.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/265d6c6e-f827-4821-9f23-74048f801a6e)
Same with the personal AI, I'm not so sure if this laptop fits the SRM theme.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/446c1de4-b9b0-4d7a-9240-3d94a8e7c9aa)
These two bottles are an abstract subtype - not meant to be seen by players.
![image](https://github.com/shiptest-ss13/Shiptest/assets/47290811/ba2a8a61-2994-421d-b47b-de528b65d066)
I think you want this welder to be an industrial welder, since other ships have an industrial welder on top of the toolboxes like this.

JimKil3 commented:
Also, you could use the wooden computer frames SolGov uses to match the rest of the ship more with the consoles

Latentish committed:
Fixes ID's and more

- Fixes the ID's by implementing the code fix for assigning a faction.
- Changes some decaling
- changes the smes configuration
- fixes a an error in the json

meemofcourse reviewed (changes requested):
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/6f69feb9-03bf-4d37-99fb-f9444a0279bb)
the Goat attacks other mobs in the area
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/12535dda-a37e-474c-8c82-2de8f5a6216b)
i feel like it would be better to use a ammo can. also, 50 x 6 is 300 rounds of ammunition. maybe halve this
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/c2838e13-8074-4e8a-a04e-15273e2f255c)
i don't think this ship warrants a crew monitoring console, and the modcomp laptop looks out of place
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/67006596-fa83-49f0-b87e-77aa53a07386)
not sure if the doubled sermon holodisk is intentional
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/5681ad31-ec10-44a5-857d-3ce1c7db4bfe)
the shotgun subtype of the guncase generates shotguns. there are also winchesters in a shotgun locker. please use the normal guncase
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/fa42f7bf-b64e-43e5-82be-a6e2b9d61034)
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/bd28aba5-6ee5-44d6-aa27-f9f19fa8d824)
you don't need this much amount of GPS
![imagen](https://github.com/shiptest-ss13/Shiptest/assets/75212565/5c9fd882-8e5a-4c81-9661-4e0e068dc7bb)
do you really need two scrubbers and two vents for this room

Latentish committed:
ELDER PATCH

- Adds surplus 357 ammo
- Rebalances the guns
- Stows away the fishing gear
- removes a pair of atmos machinery
- Removes some computers
- Removed some other gear

goober3 reviewed (approved):
gee golly... what a nice ship. i dare say, this is ready for a test merge!

goober3 commented on _maps/configs/srm_elder.json:
```diff
@@ -1,20 +1,20 @@
-	"description": "A standard issue vessel to the highest ranks of the Saint-Roumain Militia. While “standard”, this class of vessel is unique to the Montagne that owns it. Each ship is designed around a central garden consisting of plants, soil, and a tree from the owning Montagnes’ home planet. As a highly religious ascetic order, the SRM supplies each Glaive with supplies to farm, raise animals, ...
+	"map_short_name": "Elder-class",
+	"map_path": "_maps/shuttles/roumain/srm_elder.dmm",
+	"description": "A standard issue vessel to the highest ranks of the Saint-Roumain Militia. While “standard”, this class of vessel is unique to the Montagne that owns it. Each ship is designed around a central garden consisting of plants, soil, and a tree from the owning Montagnes’ home planet. As a highly religious ascetic order, the SRM supplies each Glaive with supplies to farm, raise animals, ...
```
```suggestion
	"description": "A standard issue vessel to the highest ranks of the Saint-Roumain Militia. While “standard”, this class of vessel is unique to the Montagne that owns it. Each ship is designed around a central garden consisting of plants, soil, and a tree from the owning Montagnes’ home planet. As a highly religious ascetic order, the SRM supplies each Elder-class with supplies to farm, raise animals, and perform medicine in more “natural” ways, using herbs and plants grown in house. The ship is captained by a Montagne, who oversees a team of Hunters, and Shadows apprenticing them.",
```

goober3 reviewed (changes requested):
![3305554a0809b6d6e9cf054815527c81](https://github.com/shiptest-ss13/Shiptest/assets/118859017/96cfb923-f947-428f-bcd7-434e8e6c3adc)
these guns should be more secure. put them behind windoors, perhaps.

![1de0856181d37f2b26277ebfacb76805](https://github.com/shiptest-ss13/Shiptest/assets/118859017/dd9b5a63-d373-4836-81b5-ccac9d48bd09)
these don't fit on an SRM ship.

![4c6672abf2bca2932d9edbcfa5098828](https://github.com/shiptest-ss13/Shiptest/assets/118859017/88038058-5b9f-4d70-a094-639495859a3e)
this torch clips with the window above it. pixel shift it down a little.

![e386562547ed24e8c9396ffcdedecbf7-1](https://github.com/shiptest-ss13/Shiptest/assets/118859017/0554fd55-dcd4-4fa2-8a30-356e9206fa7e)
replace this manifold with the non-visible type. vary up the flooring and the decals in this airlock a bit, we want to avoid mono-tile decal-less rooms. 

![9e4d185ed64fe2d7d8e5939865afab89](https://github.com/shiptest-ss13/Shiptest/assets/118859017/5129c7fb-f716-4a29-9211-3166c58529e2)
vary up the tiles in the engine room, too. make it practical, while still fitting the SRM's vibe.

![6490fbeba4e33ce913d9913936367710](https://github.com/shiptest-ss13/Shiptest/assets/118859017/1c42ea77-1349-4e99-8947-d24aee16ff7e)
this cargo bay seems like quite a major stylistic change to the rest of the ship. make this spot resemble the equipment room more, perhaps. 

![633e18ff8d953c475d43adb13841aa15](https://github.com/shiptest-ss13/Shiptest/assets/118859017/d3842ca6-a995-4058-b087-7863b7ba212d)
don't use this type. replace these with independent mining hardsuits, if you like. but i'm not 100% on the SRM using hardsuits at all.

![8e6ae99ba24ef1605c18ee52483a697a](https://github.com/shiptest-ss13/Shiptest/assets/118859017/4b0d82d7-b710-48d6-bafb-b09b46b1fc51)
vary up the flooring in the medbay, please. wooden flooring would suffice. 'sterile' tiles towards the operating area, perhaps. 

all in all, this is a major improvement to the glaive. good job! the style and the decalling just needs clearing up, in places. consider adding more room for cargo, also.

Skrem7 commented:
Stripper clips in the ammo box are .300 BLK, not .300 Magnum. I.E. they don't fit the Smile rifle. I'd make a new stripper clip of the same sprite, but just load it with .300 Magnum to accompany the fact that this rifle's actually in the game now.

rye-rice reviewed (approved):
357 ammo box sprite by me

Latentish committed:
Merge remote-tracking branch 'upstream/master' into SRM-Elder-Class-Montagnes-Vessel

Latentish committed:
Elder patch 2

- Changed some room tiling
- removed the 300blk strippers
- removed some soul
- added windoors to the guns
- removed the cas decks and pai

Latentish committed:
Another elder patch (wow!!!!)

- fixes the funny port
- fixes some decals

Latentish commented:
No mergies until #2618 gets merged (needs the .300 magnum ammo clip)

Zevotech reviewed (changes requested):
I'm not 100% on the torches for lighting but it is unique and pretty cool as a gimmick so I'll concede that you don't have to change it.

Plushies layer behind chairs. Also, it feels like there's too many plushies on this ship for an asetic group of religious hunters.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/7b437e1c-ce87-4da6-a22f-cfc1301d68c4)

Crates don't map onto crate shelves. Also, the fishing toolboxes already come with worms, line, hook and a fishing rod. Just make it have 3 fishing toolboxes and a fishing book or two. Additionally, having a whole crate dedicated to one disk feels wrong, maybe make the other crate hold additional ammo or possibly more disks.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/a593fb13-9267-4cff-aff9-430e19ec3ea7)

A copy of the SRM lore book (maybe include it in their backpacks in the outfit when they spawn?) and a lore holodisk feels excessive, but it's excusable. What isn't excusable is the witchhunter garb. That is chaplain armor and does not at all fit with the ship. They also do not need 3 pairs of boots. Pick a pair that best fits them and go with it. Maybe give black cowboy boots for the hunters and brown ones for the shadows. 
Remove the zippo lighters and lanterns, as those are already in the cryo room. Move the GPS devices from the cryo room into the lockers, one for each person.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/ff4f15a3-e935-476a-9157-612fd48f7605)

On the topic of equipment, remove the similarly crusty chaplain riot armor from the montagnes locker. Move the 762x38 ammo boxes to the armory if theres no gun that uses them in the locker. Same policy with the boots/lighters/adding a GPS as the previous comment.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/7e7d1e00-b4d4-4be5-8b3b-a83c99d3e55f)

Too many seeds. I'd say keep the normal ones and make them forage for the lavaland flora at least.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/d3c75e84-44ff-4b88-8ca5-396e84c5c8ab)

I have no idea why you would do this. Putting two of the same vent/scrubber next to eachother makes no sense. Either make it vent/scrubber and vent/scrubber or just make it one of each. Same for the bridge.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/cbd929ec-e94e-4d30-96ee-5a94a1ff0032)

These tiles are always lit. Change them to something else or fix the lighting issue.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/c6c0116c-8c67-4d3d-a479-293c248c69ea)


Sigh. Not fond of the tone this uses for a semi-religious tutorial book and also I'm fairly certain "clowns" would be extremely offensive to call readers in-lore. I understand you didn't write this so its fine but if possible I'd take out clowns/some of the more egregious wordings and even slate it for a full rewrite by the lore team if you can.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/9c0255f6-1d52-435f-a62d-e493f72f15a5)
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/304d0f0f-83c8-4f4d-bd1b-b38e1519f789)

Latentish committed:
elderpatchie

-moved gps'
-changed the external area to /dark
-removed the anomalous ammo boxes
-swapped the scrubbers and siphons in the entry room
-moved and replaced plushies

Latentish committed:
oop

same as last

Latentish committed:
Small changes

- Removes plushies
- Swaps out the torches in cryo for a lamp.

Latentish committed:
Merge remote-tracking branch 'upstream/master' into SRM-Elder-Class-Montagnes-Vessel

Latentish committed:
Map update

- Updates guns and ammo

Latentish committed:
Removes unneeded ammo stuff

titular

Zevotech reviewed (changes requested):
Remove the witchhunter's stuff from the Hunter's lockers. It's crufty, ugly and is still literal riot armor. This is the single hill in this review I will die on.
Also remove the excess fishing equipment. The fishing toolboxes come pre-stocked. All you need are the toolboxes and fishing manuals.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/fe9316d7-daad-45be-a172-3197ed957460)

Fix the scrubber/vent distribution in the bridge. Two of each next to eachother looks weird. Make it like how the starboard hall ones are.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/b4a46bce-36fe-4da6-b91e-1c1b65d496a4)

HP guns in the lockers have the default sprites ingame. Seems to be a bug. Ask rye.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/bc92b1ea-7b4f-4e2e-8596-7868b3b2b607)

These doors have no access restrictions. Unless this is intentional, make it so that the Hunters are the ones that can access the guns.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/7cdb1a1d-c6ff-4a38-8f0f-0609a57daebb)

Exterior hull here still uses the lit external areas, change it to the dark kind.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/fcfff4b0-4deb-4ced-abfe-6c3da3b0c112)

Just a mild gripe, these buttons don't look very good on wooden tables. I'd move them to a wall, but I wouldn't care if it was left as-is.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/6640f99e-e968-4d7a-b1af-b3c82de49c14)

This medkit is empty. If you want them to have a filled one, use the /standard type of it.
![image](https://github.com/shiptest-ss13/Shiptest/assets/95449138/1766fb3d-cfe3-4b74-a1ca-37e01739bae1)

Latentish committed:
fixes flaming bolt and minor changes

- fixes flaming bolt using the wrong sprite
- fixes lockers
- changes the button on the bridge
- nerfs fishing gear
- changes up some atmos stuff

Latentish commented:
also removed witchhunter gear and the cultpack, and changed the external area to /dark

meemofcourse reviewed (approved):
me when i take 17 trickwine and start seeing the bull
